uni/lib/view/restaurant/restaurant_page_view.dart

```diff
@@ -16,11 +16,22 @@ class RestaurantPageView extends StatefulWidget {
   const RestaurantPageView({Key? key}) : super(key: key);
 
   @override
-  State<StatefulWidget> createState() => _CanteenPageState();
+  State<StatefulWidget> createState() => _CantinePageState();
 }
 
-class _CanteenPageState extends GeneralPageViewState<RestaurantPageView>
+class _CantinePageState extends GeneralPageViewState<RestaurantPageView>
     with SingleTickerProviderStateMixin {
+
+  final List<DayOfWeek> daysOfTheWeek = [
```

Comment:
Why did this come back? It should not be here.

Reply:
`createRowFromRestaurant(context, restaurants[index], daysOfTheWeek[offset])` -> this argument